Fix failure to link python* packages to the lib.

http://git.pld-linux.org/?p=packages/qscintilla2.git;a=blob;f=qscintilla2-link.patch;h=999b2293d0eb2d901aff3ae6d249c0f5e39a5865;hb=23821e39e01f15e379efc9d00c9e0b4dd9919ef0

--- QScintilla_gpl-2.10.3/Python/configure.py.orig_	2018-03-19 17:27:18.661667989 -0300
+++ QScintilla_gpl-2.10.3/Python/configure.py	2018-03-19 18:56:30.697768397 -0300
@@ -1553,7 +1557,10 @@ INSTALLS += sip
 
     libs = qmake_config.get('LIBS')
     if libs:
-        pro.write('LIBS += %s\n' % libs)
+        if target_config.pyqt_package == 'PyQt5':
+            pro.write('LIBS += %s -lqscintilla2_qt5\n' % libs)
+        else:
+            pro.write('LIBS += %s -lqscintilla2_qt4\n' % libs)
 
     if not opts.static:
         dylib = module_config.get_mac_wrapped_library_file(target_config)
